jaeger-core/src/main/java/com/uber/jaeger/Tracer.java

```diff
@@ -449,6 +453,10 @@ public Scope startActive(boolean finishSpanOnClose) {
     private ScopeManager scopeManager = new ThreadLocalScopeManager();
     private BaggageRestrictionManager baggageRestrictionManager = new DefaultBaggageRestrictionManager();
 
+    public Builder() {
```

Comment:
maybe `Tracer.builder(serviceName)` instead? What are the conventions in Java these days?  I personally prefer static methods to constructors.